docs/sources/enterprise/activate-license.md

```diff
@@ -1,4 +1,9 @@
-
++++
+title = "License Activation"
```

Comment:
License activation

(Use sentence style for headings)

Reply:
Updated